### ISSUE TITLE
Version 1.3.0 doesn't work with Java 7

### ISSUE THREAD

**Issue by jakubkrolewski** [allegro/axion-release-plugin#94]
```
$javac -version
javac 1.7.0_25
```
Build failure:
```
FAILURE: Build failed with an exception. 	
* What went wrong:
java.lang.UnsupportedClassVersionError: pl/allegro/tech/build/axion/release/ReleasePlugin : Unsupported major.minor version 52.0
> pl/allegro/tech/build/axion/release/ReleasePlugin : Unsupported major.minor version 52.0
```

Version 1.2.4 works fine.

**Comment by adamdubiel** [comment]
This must have been deployment env failure. Unfortunately there is no way to remove already published artifact, so fix will come in 1.3.1.

**Comment by szpak** [comment]
> Unfortunately there is no way to remove already published artifact, so fix will come in 1.3.1.

IMHO in general it is a very good thing for public repositories. The artifact(s) could be already downloaded/cached and uploading modified artifacts with the same name/version could generate strange issues for those users.

**Comment by adamdubiel** [comment]
I agree and would not want this to change :)

**Comment by philbeiler** [comment]
Will you have the opportunity to publish this patched version to the Maven central repository?  It does not appear that 3.0.0 was ever published.  Thanks.

**Comment by adamdubiel** [comment]
@philbeiler - new Gradle plugin deployment destroyed the Central sync.. if you still need newest versions from Central i will have to reconfigure it or go with double deployment.

**Comment by philbeiler** [comment]
That is your call. I hope that I'm not the only person stuck using the Maven repositories.  It would be nice to pull it from there, otherwise I will have to build it from source. Just let me know your plans. Thanks again for the great work!

**Comment by adamdubiel** [comment]
@jakubkrolewski - this issue has been fixed in 1.3.1, artifacts are Java7 compatible.
@philbeiler - i also added Central integration (i think it's the 3rd time i'm doing this :) ), 1.3.1 is on it's way to Central at this moment
